### ISSUE TITLE
XDL PackageLogsStream may be able to handle more types of packager events

### ISSUE THREAD

**Issue by vjpr** [expo/expo-cli#206]
https://github.com/expo/xdl/blob/cf53633e47bdbc56a7a6549a5838f15c6973f4db/src/logs/PackagerLogsStream.js#L96-L119

```js
  _handlePackagerEvent = (chunk: any) => {
    let { msg } = chunk;


    if (!msg.type) {
      return;
    } else if (msg.type && msg.type.match(/^bundle_/)) {
      this._handleBundleTransformEvent(chunk);
      return;
    }


    if (msg.type === 'dep_graph_loading') {
      chunk.msg = 'Loading dependency graph.'; // doesn't seem important to log this
    } else if (msg.type == 'dep_graph_loaded') {
      chunk.msg = 'Dependency graph loaded.'; // doesn't seem important to log this
    } else if (msg.type === 'transform_cache_reset') {
      chunk.msg = 'Your JavaScript transform cache is empty, rebuilding (this may take a minute).';
    } else if (msg.type === 'initialize_packager_started') {
      chunk.msg = `Running packager on port ${msg.port}.`;
    } else {
      chunk.msg = '';
    }


    this._enqueueAppendLogChunk(chunk);
  };
```

It matches only 4 event types and if it doesn't match it removes the error.

I had to manually add a `console.log` statement to find the reason my build wasn't working.

```
   { type: 'worker_stderr_chunk',
     chunk: 'module.js:538\n    throw err;\n    ^\n\nError: Cannot find module \'babel-template\'\n    at Function.Module._resolveFilename (module.js:536:15)\n    at Function.Module._load (module.js:466:25)\n    at Module.require (module.js:579:17)\n    at require (internal/module.js:11:18)\n    at Object.<anonymous> (/Users/Vaughan/dev-live/photo-booth-ios-crna/node_modules/.registry.npmjs.org/metro-pnpm/0.24.7-vjpr.1/node_modules/metro-pnpm/src/ModuleGraph/worker/collectDependencies.js:15:23)\n    at Module._compile (module.js:635:30)\n    at Object.Module._extensions..js (module.js:646:10)\n    at Module.load (module.js:554:32)\n    at tryModuleLoad (module.js:497:12)\n    at Function.Module._load (module.js:489:3)\n' } }
```

Why not just set `chunk.msg = msg.chunk || ''` instead of `chunk.msg = ''`?

Is there a verbose mode?

**Comment by fson** [comment]
Looks like we now explicitly handle all the error types Metro may produce ([the `ReportableEvent` type](https://github.com/facebook/metro/blob/92f8e5deee2fb574ccf68d7ce4de5fecf7477df6/packages/metro/src/lib/reporting.js#L32) in Metro lists all the event types.

However, we explicitly ignore a few of them here: https://github.com/expo/expo-cli/blob/400b7486da7f04c4b3d9050df96d59a18a530582/packages/xdl/src/logs/PackagerLogsStream.js#L259-L265

If some of these should not be ignored (e.g. `worker_stderr_chunk`? Does it include any information that is not reported as `bundling_error`?), could you send a PR to make PackagerLogsStream report them? It should just be tested carefully to make sure we're not adding unnecessary noise to the console.

**Comment by fson** [comment]
Fixed in 91cef486b46dde4f0459dad777b5f0155026e17a and 92bdc66a868c40e865124586d8c50282f357d756.
